documentation/api/processes-env.md

```diff
@@ -0,0 +1,90 @@
+# Processes - Get Environment
+
+Gets the environment block of a specified process.
+
+```http
+GET https://localhost:52323/processes/{pid}/env
+```
+
+or 
+
+```http
+GET https://localhost:52323/processes/{uid}/env
+```
+
```

Comment:
Consider adding a note to help people understand when they should use `pid` and when they should use `uid`, or perhaps to repeat the warning that `uid` will be empty for a .NET Core 3.1 application.

Reply:
I think I'll do the same as the other: centralized doc area with links from each relevant page.